### DISCUSSION TITLE
Can I somehow split `dylib`s / `Frameworks` for different archs in tauri.conf.json > bundle > macOS > frameworks?

### DISCUSSION THREAD

**Discussion by timleg002** [tauri-apps/tauri#8969] [Q&A]
I need to include a few `dylib`s in my project, I need to support both aarch64 and x86_64, and a fat binary doesn't seem like a ideal choice space-wise. Is there any approach to this? Thank you.

**Comment by FabianLars** [comment]
i'd say you currently have to use the `--config` flag for this. So for example you'll have a tauri.x86.json file that contains only the frameworks configs pointing to the respective architecture. Then you build it with `tauri build --config path/to/tauri.x86.json`.

In the future i hope we can extend [platform specfic configs](https://tauri.app/v1/api/config#platform-specific-configuration) to also include the arch (optionally)

**Reply by timleg002** [reply]
Thank you for your help. I think splitting the configs according to arch according to target triple would be better. E.g. `tauri.aarch64-apple-darwin.conf.json`, maybe optionally using the alternative `tauri.macos.conf.json` as a catch-all for all macOS archs. I'd also appreciate more freedom in splitting the externalBins however I want, for example splitting them in directories, instead of appending a suffix to the file names.
